common/app/staticpages/StaticPages.scala

```diff
@@ -1,8 +1,15 @@
 package staticpages
 
-import model.{DotcomContentType, MetaData, SectionId, SimplePage}
+import model.{DotcomContentType, MetaData, SectionId, SimplePage, StandalonePage}
 import services.newsletters.{GroupedNewslettersResponse, NewsletterResponse}
 
+case class NewsletterRoundupPage(
+    metadata: MetaData,
+    groupedNewsletterResponses: List[(String, List[NewsletterResponse])],
+) extends StandalonePage {
+  val groupedNewslettersResponses = groupedNewsletterResponses
+}
+
```

Comment:
@jfsoul I added this so that we don't need the newsletter data to be in the metadata. Metadata comes from the same place it did previously for `SimplePage`, but then we can pass this through as a second argument.